lib/solidus_paypal_braintree/testing_support/factories.rb

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
```

Comment:
Should we move the existing factories into here @MinasMazar?
https://github.com/solidusio/solidus_paypal_braintree/blob/master/lib/solidus_paypal_braintree/factories.rb

Reply:
You're right! Updated [here](https://github.com/solidusio/solidus_paypal_braintree/pull/289/commits/276386929c928efa68e1b059a6df184f13874ba0).